### PR TITLE
Remove step

### PR DESCRIPTION
**Pull request by oleurud**
Solves: https://github.com/CartoDB/Windshaft/issues/644

First, I have created a PR for each module and everyone has been tested. Now, I am merging all the PRs in one

### CODE REVIEW

File: lib/windshaft/renderers/mapnik/factory.js [hunk 1]
```diff
@@ -159,8 +157,7 @@ MapnikFactory.prototype.defineExpectedParams = function (params) {
 };
 
 MapnikFactory.prototype.getRenderer = function (mapConfig, format, options, callback) {
-    var self = this;
-
+    /*jshint maxcomplexity:10 */
```
Comment (oleurud) [inline lib/windshaft/renderers/mapnik/factory.js]: I should make a new refactor...

File: lib/windshaft/renderers/mapnik/factory.js [hunk 2]
```diff
@@ -424,6 +410,20 @@ function isRasterColumnType(geomColumnType) {
     return geomColumnType === COLUMN_TYPE_RASTER;
 }
 
+function getMMLBuilder(mapConfig, grainstoreOptions, params, format) {
```
Comment (dgaubert) [inline lib/windshaft/renderers/mapnik/factory.js]: ❤️

**Comment by javitonino** [comment]
@oleurud status report? Can this be deployed? Who is going to do it? Just asking for tracking purposes, since this issue is currently unassigned in the Kanban, so I'm not sure who should make the next step.

**Comment by oleurud** [comment]
The idea was to deploy yesterday, but checking everything was fine, I found I forgot a module with step :scream: So we need to make the same as the rest of the modules and finally merge in this PR (really, we could make it in this branch directly)

About the assignation, I do not usually assign a PR to me even though I'm doing it, should I do it? My idea is to continue working when `Dataservices/oauth` allows it

**Comment by javitonino** [comment]
The basic idea about assignment:
- If it's unassigned, first person to come, will take it. Since it's in the deploy column, the first person to come should deploy this if possible.
- If it's assigned, it's the person working on it.
  - It only matters on the board. While you are developing it, if it's not in the board, leave it unassigned if you want.
  - If you are CRing/Accepting, you should have it assigned (to avoid other people from CRing it and doubling the effort)
  - If your CR requires changes, assign it back to the original developer so he knows it's his turn to reply/make changes.

Basically, the person assigned should be the one taking the next step. If it's unassigned, first come, first served. From what you say, this PR should be assigned to you for the moment, since you intend to finish it.

**Comment by oleurud** [comment]
It will be blocked until https://github.com/CartoDB/Windshaft/pull/663 will be merged.

Before moving it to acceptance:
- [ ] merge master into it
- [ ] remove `step` from dependencies (it will be in devDependencies because it will still be used in tests)
- [ ] Prepare release: NEWS and version

**Comment by oleurud** [comment]
All the branches are merged and tested. Now it is time for the last CR. New things:
- Last module with step removed merged
- merge master into this
- remove step from dependencies (it will be in devDependencies because it will still be used in tests)
